docs/src/pages/components/slider/CustomizedSlider.js

```diff
@@ -42,14 +42,12 @@ const marks = [
 const IOSSlider = styled(Slider)(({ theme }) => ({
   color: theme.palette.mode === 'dark' ? '#3880ff' : '#3880ff',
   height: 2,
-  padding: '15px 0',
+  margin: '15px 0',
```

Comment:
This looks wrong. margin is not part of the draggable touch zone.

Reply:
revert back to use padding.